composer.json

```diff
@@ -18,7 +18,9 @@
     "pimcore/customer-management-framework-bundle": "^3.0",
     "pimcore/pimcore": "10.0.x-dev as 6.8.x-dev",
     "pimcore/web2print-tools-bundle": "^4.0",
-    "pimcore/data-hub": "^1.0"
+    "pimcore/data-hub": "^1.0",
+    "pimcore/payment-provider-unzer": "^1.0",
+    "pimcore/payment-provider-datatrans": "^1.0"
```

Comment:
since datatrans is not implemented, let's remove it completely ... also from config

Reply:
removed